rare/components/tabs/games/cloud_save_utils.py

```diff
@@ -170,16 +172,16 @@ def sync_before_launch_game(self, app_name, ignore_settings=False) -> int:
                 logger.info(f"Set save path of {igame.title} to {savepath}")
             elif not ignore_settings:  # sync on startup
                 if (
-                        QMessageBox.question(
-                            None,
-                            "Warning",
-                            self.tr(
-                                "Could not compute cloud save path. Please set it in Game settings manually. \nDo you want to launch {} anyway?"
-                            ).format(igame.title),
-                            QMessageBox.Yes | QMessageBox.No,
-                            QMessageBox.No,
-                        )
-                        == QMessageBox.Yes
+                    QMessageBox.question(
+                        None,
+                        "Warning",
+                        self.tr(
+                            f"Could not compute cloud save path. Please set it in Game settings manually. \nDo you want to launch {igame.title} anyway?"
```

Comment:
Hm. Sure that this preserves the translation being dynamic? Afaik f-strings are eagerly evaluated, so that might end up feeding a completely other string to `self.tr` — causing no translation to be found for anything. But maybe I'm wrong and `self.tr` uses stuff I don't know of.

Reply:
After looking at this https://stackoverflow.com/questions/65499544/what-is-the-equivalent-of-qstring-text-tr1-argqstringformat-touppe/65499730#65499730 im pretty sure that we are able to use f-strings, so i guess we are fine.

Reply:
Yes, you are right. After testing, i can verify that f-strings dont work. I will push commits resolving all those f-strings in tr()

Reply:
Yes, you are right. After testing, i can verify that f-strings dont work. I will push commits resolving every f-string in tr() functions